2-mandatory/6-journey-planner.js

```diff
@@ -20,8 +20,8 @@
 function checkCodeIsThere(stringText) {
   let magicWord = "code";
   //edit code below
-  if (stringText) {
-    return stringText;
+  if (stringText.indexOf(magicWord)) {
+    return stringText.index;
```

Comment:
this is very close, but you are using `.index` and this isn't actually a method on stringText, so it is returning undefined. You want to instead be returning `stringText.indexOf(magicWord)` in your return statement